common/djangoapps/enrollment/management/commands/enroll_user_in_course.py

```diff
@@ -0,0 +1,56 @@
+"""
+Management command for enrolling a user into a course via the enrollment api
+"""
+from optparse import make_option
```

Comment:
I had looked at argparse and was only getting hits for it in paver at the time. So, thanks for the reference. I may not have time to address it but lemme see.